onnx/backend/test/case/node/unique.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Unique(Base):
+
+    @staticmethod
+    def export_without_axis():  # type: () -> None
+        x = np.array([2.0, 1.0, 1.0, 3.0, 4.0, 3.0], dtype=np.float32)
+
+        node_sorted = onnx.helper.make_node(
+            'Unique',
+            inputs=['X'],
+            outputs=['Y', 'indices', 'inverse_indices', 'counts']
+        )
+        y, indices, inverse_indices, counts = np.unique(x, True, True, True)
+        expect(node_sorted, inputs=[x], outputs=[y, indices, inverse_indices, counts], name='test_unique_sorted_without_axis')
+
+        node_not_sorted = onnx.helper.make_node(
+            'Unique',
+            inputs=['X'],
+            outputs=['Y', 'indices', 'inverse_indices', 'counts'],
+            sorted=0
+        )
+        # numpy unique does not retain original order (it sorts the output unique values)
```

Comment:
Is it possible to add a python implementation for this case? E.g., can we use some sort-routine on inverse-indices to permute all the outputs to the required order?